src/test/java/com/google/plantasy/TimingPropertiesTest.java

```diff
@@ -46,139 +46,133 @@ public void tearDown() {
         Assert.assertEquals(null, actual);
     }
 
     @Test 
     //Tests if a valid timestamp object is returned
     public void getQuizTimestampProperty_validParamters() {
-        QuizTimingPropertiesUtils timing_properties_test = new QuizTimingPropertiesUtils();
-
         Entity user = new Entity("user");
         user.setProperty("quiz_timestamp", 1594309443653L);
         user.setProperty("userID", "abc");
 
         DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
         datastore.put(user);
 
-        long actual = timing_properties_test.getQuizTimestampProperty("user", "userID", user.getProperty("userID").toString(), datastore);
+        long actual = QuizTimingPropertiesUtils.getQuizTimestampProperty("user", "userID", user.getProperty("userID").toString(), datastore);
         Assert.assertEquals(1594309443653L, actual);
     }
 
     @Test(expected = IllegalArgumentException.class)
     //Tests for paramters being null
     public void userTookQuiz_nullParameters() {
-        QuizTimingPropertiesUtils timing_properties_test = new QuizTimingPropertiesUtils();
-
         Long user_quiz_time = null;
-        Long current_quiz_time = null;
-
-        boolean actual = timing_properties_test.userTookQuiz(user_quiz_time, current_quiz_time);
+        
+        boolean actual = QuizTimingPropertiesUtils.userTookQuiz(user_quiz_time);
+        Assert.assertEquals(false, actual);
     }
 
     @Test
-    //Tests valid string, timestamp paramters 
-    public void userTookQuiz_validParameters() {
-        QuizTimingPropertiesUtils timing_properties_test = new QuizTimingPropertiesUtils();
-
+    //Tests valid string, timestamp paramters for false test case
+    public void userTookQuiz_validParametersFalse() {        
         Entity user = new Entity("user");
-        Entity game_entity = new Entity("Game");
-        Game game = new Game(game_entity);
-
         DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
 
         user.setProperty("quiz_timestamp", 1594309443653L);
         user.setProperty("userID", "abc");
         datastore.put(user);
+ 
+        Long user_quiz_time = QuizTimingPropertiesUtils.getQuizTimestampProperty("user", "userID", user.getProperty("userID").toString(), datastore);
+         
+        boolean actual = QuizTimingPropertiesUtils.userTookQuiz(user_quiz_time);
+        Assert.assertEquals(false, actual);
+    }
 
-        game.setQuizTimestamp(1594309443660L);
-        datastore.put(game.getGameEntity());
+    @Test
+    //Tests valid string, timestamp parameters for true test case
+    public void userTookQuiz_validParametersTrue(){
+        Entity user = new Entity("user");
+        DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
+
+        user.setProperty("quiz_timestamp", System.currentTimeMillis());
```

Comment:
Typically, when testing, we don't want to use dynamic values like this. However, as you discovered, we need to do that here. 

I would suggest leaving a comment stating not to change this line and the reason why. Otherwise, someone might come in here, say "hmm this should be some hard coded value like in the function above", change it, run the test, discover it works (because maybe they used today's timestamp), then tomorrow it breaks again.